addons/l10n_cl/views/account_move_view.xml

```diff
@@ -15,6 +15,31 @@
         </field>
     </record>
 
+    <record id="view_latam_form_inherit_l10n_cl" model="ir.ui.view">
+        <field name="name">account.move.latam.form.inherit.l10n.cl</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="l10n_latam_invoice_document.view_move_form"/>
+        <field name="arch" type="xml">
+            <field name="l10n_latam_document_number" position="attributes">
+                <attribute name="attrs">
+                    {'invisible': [
+						'|', ('l10n_latam_use_documents', '=', False),
+                    	     ('l10n_latam_manual_document_number', '=', False),
+                    	'|', '|', ('l10n_latam_use_documents', '=', False),
+                    	          ('highest_name', '!=', False),
+                    	          ('state', '!=', 'draft'),
+                    	'|', '|', '|', ('l10n_latam_use_documents', '=', False),
+                    	               ('posted_before', '=', False),
+                    	               ('state', '!=', 'draft'),
+                    	               ('country_code', '!=', 'CL')
+                    ],
+                    'readonly': [('posted_before', '=', True), ('state', '!=', 'draft'), ('country_code', '!=', 'CL')]
```

Comment:
As the readonly is True only when the state is not draft, I don't know why we add the extra condition country_code != 'CL'

Also, why do we remove the "required" one or is it still kept magically?

Reply:
We cannot remove the country_code condition due to the posted_before field.

Ups! I forgot the required attribute 🤓

Reply:
@suv-bmya I don't understand.  In order for it to be readonly you should not be in draft state anyways.  I think you should only modify it when the state is in draft anyways, so I don't see why you need the extra condition.

Reply:
Nothing! You're right...We don't need this condition.